documentation/secure_storage_sql.md

```diff
@@ -0,0 +1,77 @@
+# SQL DB Secure Storage
+
+## Introduction
+
+This document describes the SQL DB secure storage in OP-TEE, which is enabled
```

Comment:
Hmm... I don't think so: http://www.writersdigest.com/online-editor/which-vs-that
Maybe a native speaker can confirm or infirm?